onnx/onnx.proto

```diff
@@ -724,6 +732,8 @@ message TensorProto {
   // Complex128 elements must be written as two consecutive DOUBLE values, real component first.
   // Boolean type MUST be written one byte per tensor element (00000001 for true, 00000000 for false).
   // uint4 and int4 values must be packed to 4bitx2, the first element is stored in the 4 LSB and the second element is stored in the 4 MSB.
+  // uint2 and int2 values must be packed to 2bitx4, with elements packed from LSB to MSB in a single byte as: (x3 << 6) | (x2 << 4) | (x1 << 2) | x0
+  // where x0, x1, x2, x3 are consecutive elements.
```

Comment:
```suggestion
  // uint2 and int2 values must be packed to 2bitx4, with elements packed from LSB to MSB in a single byte as: x0 | (x1 << 2) | (x2 << 4) | (x3 << 6) 
  // where x0, x1, x2, x3 are consecutive elements.
```
(minor) Listing left-to-right would be consistent with the `x0, x1, x2, x3` order, with uint4 order (which lists LSB then MSB), and with the code below `array_flat &= 0x03; array_flat[1::4] <<= 2; array_flat[2::4] <<= 4...`.